pkg/cloudprovider/aws/instancetypes.go

```diff
@@ -189,3 +190,11 @@ func (p *InstanceTypeProvider) CacheUnavailable(ctx context.Context, fleetErr *e
 func UnavailableOfferingsCacheKey(capacityType string, instanceType string, zone string) string {
 	return fmt.Sprintf("%s:%s:%s", capacityType, instanceType, zone)
 }
+
+func InstanceTypesCacheKey(provider *v1alpha1.AWS) string {
+	providerHash, err := hashstructure.Hash(provider, hashstructure.FormatV2, nil)
+	if err != nil {
+		panic(fmt.Sprintf("hashing provider spec, %s", err))
+	}
+	return fmt.Sprintf("types:%d", providerHash)
```

Comment:
Don't try to make this human readable
```suggestion
	return fmt.Sprintf("%d", providerHash)
```

Reply:
Fixed.